Makefile

```diff
@@ -38,4 +38,8 @@ vendor:
 	$(GO) mod vendor
 	$(GO) mod verify
 
-.PHONY: build test lint vendor coverage codecov
+clean:
+	make -C crit/ clean
+	make -C test/ clean
```

Comment:
we should use `$(MAKE)` for [recursive invocations](https://www.gnu.org/software/make/manual/html_node/Recursion.html): https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html#MAKE-Variable